.github/workflows/go.yml

```diff
@@ -0,0 +1,27 @@
+name: CI Go
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+
+    - name: Set up golang
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.16
+
+    - name: Build binaries
```

Comment:
The PR looks good to me but what do you think about adding a format check here?

In a separate PR, maybe we could add workflows for e2e tests as well. Not a blocker in anyway for this PR!

Reply:
format like in `go fmt`? sure thing

Reply:
e2e + artifacts creation + push will toatally come in followup PRs